crates/core/src/subscription/module_subscription_manager.rs

```diff
@@ -0,0 +1,389 @@
+use super::execution_unit::{ExecutionUnit, QueryHash};
+use crate::client::messages::{CachedMessage, TransactionUpdateMessage};
+use crate::client::{ClientConnectionSender, DataMessage, Protocol};
+use crate::db::relational_db::RelationalDB;
+use crate::error::DBError;
+use crate::execution_context::ExecutionContext;
+use crate::host::module_host::{DatabaseTableUpdate, DatabaseUpdate, ModuleEvent};
+use smallvec::SmallVec;
+use spacetimedb_lib::identity::AuthCtx;
+use spacetimedb_lib::Identity;
+use spacetimedb_primitives::TableId;
+use std::collections::{HashMap, HashSet};
+use std::sync::Arc;
+
+type Id = Identity;
```

Comment:
I think this alias makes things more opaque.

Reply:
I think you're right. I will remove this alias.